### PR TITLE
Add severity level debug in logging

### PR DESCRIPTION
**Pull request by NAVRockClimber**
Added a serverity level debug to the logging function due logging compiler errors as error will cause ALOps to cut the log output.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,4 +6,5 @@
 - Add enablePremium handling
 - Handle restart gracefully
 - DevOps artifacts were ignored
-- Support custom scripts
\ No newline at end of file
+- Support custom scripts
+- Add severity level "debug" to logging (see [1232](https://dev.azure.com/cc-ppi/General/_workitems/edit/1232))
```
Comment (tfenster) [inline CHANGELOG.md:10]: Can you please move this to the top? We have no real versioning here, but the order should be newest to oldest

File: artifacts/ArtifactHandling/Add-ArtifactsLog.ps1 [hunk 2]
```diff
@@ -43,8 +44,9 @@ function Add-ArtifactsLog {
         if (! $message) { Write-Host "$info "; return }
         switch ($severity) {
             "Info"  { foreach ($m in "$message".Trim().Split([System.Environment]::NewLine)) { if ($m) { Write-Host "$info $($m.trim())" } } }
-            "Warn"  { foreach ($m in "$message".Trim().Split([System.Environment]::NewLine)) { if ($m) { Write-Host "$info $($m.trim())" -f Yellow } } }
+            "Warn"  { foreach ($m in "$message".Trim().Split([System.Environment]::NewLine)) { if ($m) { Write-Host "$info $($m.trim())" -f Yellow } } }
```
Comment (tfenster) [inline artifacts/ArtifactHandling/Add-ArtifactsLog.ps1:47]: do we need this whitespace?

**Review by tfenster — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (NAVRockClimber) [reply CHANGELOG.md]: I moved this to the top

File: artifacts/ArtifactHandling/Add-ArtifactsLog.ps1 (hunk 2 above)
Comment (NAVRockClimber) [reply artifacts/ArtifactHandling/Add-ArtifactsLog.ps1]: I removed the whitespaces

**Review by tfenster — APPROVED** [review]

**Review by megel — APPROVED** [review]

**Pull request merged**